app/views/spree/users/_authorised_shops.html.haml

```diff
@@ -7,10 +7,11 @@
     %th= t(".allow_charges?")
   %tr.customer{ id: "customer{{ customer.id }}", ng: { repeat: "customer in customers" } }
     %td.shop{ ng: { bind: 'shopsByID[customer.enterprise_id].name' } }
-    %td.allow_charges
+    %td.allow_charges{tooltip: "{{ hasOneDefaultSavedCards() ? null : ('spree.users.authorised_shops.no_default_saved_cards_tooltip' | t) }}",}
```

Comment:
Since this is a normal view, you could use the Ruby method:
```suggestion
    %td.allow_charges{tooltip: "{{ hasOneDefaultSavedCards() ? null : '#{t(\".no_default_saved_cards_tooltip\")' }}",}
```

Upside: lazy lookup (shorter key).
Downside: additional quotes.

:shrug: